packages/api/src/store/cache.ts

```diff
@@ -0,0 +1,23 @@
+import NodeCache from "node-cache";
+
+export const cache = new NodeCache({ stdTTL: 60, maxKeys: 1000 });
```

Comment:
Should we document it somewhere that changes to user access may take 1 minute to have effect.

Reply:
I think a little read after write inconsistency is somewhat expected in APIs like this (and we also already had _some_ of it due to database read replicas). Since we are trying to only cache objects that are mostly immutable right now (users, object stores, api&access keys, etc) I'd rather keep that behavior implicit for now. 

Wouldn't oppose to document it if anyone has a strong opinion about it though.